app/src/androidTest/java/com/github/onlynotesswent/ui/EndToEndTest.kt

```diff
@@ -273,20 +267,22 @@ class EndToEndTest {
     composeTestRule.onNodeWithTag("createNote").performClick()
     composeTestRule.onNodeWithTag("confirmNoteAction").assertIsDisplayed()
     composeTestRule.onNodeWithTag("inputNoteName").performTextInput(testNote.title)
-    composeTestRule.onNodeWithTag("visibilityButton").performClick()
-    composeTestRule.onNodeWithTag("item--Public").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("item--Friends Only").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("item--Private").assertIsDisplayed().performClick()
+    composeTestRule.onNodeWithTag("currentVisibilityOption").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("previousVisibility").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("nextVisibility").performClick()
     composeTestRule.onNodeWithTag("confirmNoteAction").assertIsDisplayed()
     composeTestRule.onNodeWithTag("confirmNoteAction").performClick()
 
     // Modify the note title and save the changes
+    Log.d("selected note", noteViewModel.selectedNote.value.toString())
```

Comment:
Probably you forgot to delete the logs for this test

Reply:
Yes thank you for noticing